modules/cudn-status-conditions.adoc

```diff
@@ -80,3 +80,157 @@ h|Message
+h|Reason
+h|Message
+h|Resolution
+
+|The name of the physical network is not set.
+|`spec.network.localnet.physicalNetworkName: Required value`
+|You must set the `physicalNetworkName` field.
+
+|The name of the physical network does not meet minimum length requirements.
+|`spec.network.localnet.physicalNetworkName in body should be at least 1 chars long`
+|You must set physical network name to be at least one character in length.
+
+|The name of the physical network exceeds the maximum character limit of 253.
+|`spec.network.localnet.physicalNetworkName: Too long: may not be more than 253 bytes`
+|You must set physical network name to not exceed the 253 character in length.
+
+|The name of the physical network must not contain `,` or `:`.
+|`physicalNetworkName cannot contain "," or ":" characters`.
+|You must remove the `,` or `:` from the physical network name.
+|===
+
+.Invalid `role` scenarios types for the `ClusterUserDefinedNetwork` CR
+[cols="2a,2a,3a,3a",options="header"]
+|===
+
+|Condition type
+3+|Reason, Message, Resolution
+
+.6+|`role unset` or `role is primary`
+3+|One of the following messages is returned when the `spec.network.localnet.role` is set incorrect:
+h|Reason
+h|Message
+h|Resolution
+
+|The `role` field must be set for your localnet topology.
+|`spec.network.localnet.role: Required value`
+|You must set the `role` field.
+
+|`Primary` is not a supported value for the Localnet topology.
+|`spec.network.localnet.role: Unsupported value: "Primary": supported values: "Secondary"`
+|You must set the `role` field for your Localnet topology to `Secondary`-the accepted value.
+|===
+
+.Invalid `subnets` and `ipam` scenarios types for the `ClusterUserDefinedNetwork` CR
+[cols="2a,2a,3a,3a",options="header"]
+|===
+
+|Condition type
+3+|Reason, Message, Resolution
+
+.9+|`LocalnetInvalidSubnets`
+3+|One of the following messages is returned when either the `spec.network.localnet.subnets` or `spec.network.localnet.ipam` is set incorrect:
+h|Reason
+h|Message
+h|Resolution
+
+|The optional fields, `subnets` and `ipam.mode`, have to be set together.
+|`Subnets is required with ipam.mode is Enabled or unset, and forbidden otherwise`
+|You must set the `subnets` field unless the `spec.network.localnet.ipam.mode` is explicitly disabled.
+
+|The `spec.network.localnet.subnets` must have an acceptable value when using this optional field.
+|`The ClusterUserDefinedNetwork "localnet-empty-subnets-fail" is invalid: spec.network.localnet.subnets: Invalid value: 0: spec.network.localnet.subnets in body should have at least 1 items`
+|You must set an acceptable value for `spec.network.localnet.subnets`. Acceptable values are IPv4 and IPv6 Classless Inter-Domain Routing (CIDR) ranges that do not overlap with any CIDR ranges used by {product-title}.
+
+|The `subnet` field must be set when using the optional `spec.network.localnet.excludeSubnets` field.
```

Comment:
@asood-rh can we include some wording here rather than adding a whole other row. The goal isn't to catalog every error message but the ones that would be most useful. What about adding `You must set the value for the excludeSubnets field to be within the subnets field .` For example, a `subnets` value of `192.168.100.0/24` and an `excludeSubnets` value of `192.168.200.1/32` is invalid.

Reply:
included